File: app/src/main/java/com/qwert2603/spenddemo/model/Kind.java
package com.qwert2603.spenddemo.model;

import java.sql.ResultSet;
import java.sql.SQLException;

public class Kind {
    private String mKind;

    public Kind(ResultSet resultSet) throws SQLException {
        mKind = resultSet.getString("kind");
    }

    public String getKind() {
        return mKind;
    }

    public void setKind(String kind) {
        mKind = kind;
    }
}

File: spend/src/main/java/com/qwert2603/spenddemo/model/entity/Kind.kt
package com.qwert2603.spenddemo.model.entity

import com.qwert2603.andrlib.model.IdentifiableLong
import com.qwert2603.andrlib.model.hashCodeLong
import java.util.*

data class Kind(
        val kind: String,
        val lastPrice: Int,
        val lastDate: Date
) : IdentifiableLong {
    override val id = kind.hashCodeLong()
}
